CMakeLists.txt

```diff
@@ -465,6 +465,11 @@ else()
   message(WARNING "C++ charconv is unusable!\nWill use own implementation.")
 endif()
 
+try_compile(HAVE_STRTOX_L ${CMAKE_BINARY_DIR} "${CMAKE_SOURCE_DIR}/checks/strtox_l.cpp")
```

Comment:
These functions are part of POSIX and supported since MSVC 2015 at least, so no need for additional checks.